ClavaLaraApi/src-lara-clava/clava/clava/code/DecomposeResult.js

```diff
@@ -1,11 +1,22 @@
 class DecomposeResult {
+  constructor(precedingStmts, $resultExpr, succeedingStmts = []) {
```

Comment:
Should it still be called `$resultExpr`? Below there is a place where it is called `$innerExpr`. Other name suggestions would be `$simplifiedExpr`, `$decomposedExpr`.

Reply:
However, we have to consider that several places might already use $resultExpr, and if it would be worth the effort of changing the name